values.yaml

```diff
@@ -1483,6 +1483,14 @@ connectInject:
       memory: "50Mi"
       cpu: "50m"
 
+  # Sets the failurePolicy for the mutating webhook. By default this will cause pods to fail scheduling while the webhook
```

Comment:
```suggestion
  # Sets the failurePolicy for the mutating webhook. By default this will cause pods not part of the consul installation to fail scheduling while the webhook
```